websites/W/Wolfy/presence.ts

```diff
@@ -27,6 +26,89 @@ function getTimestamps(audioTime: number, audioDuration: string): number[] {
 	];
 }
 
+async function addJoinGameButton(presenceData: PresenceData, gameId: string) {
+	if (!(await presence.getSetting("joinGameButton"))) return;
+	if (!presenceData.buttons) {
+		presenceData.buttons = [
+			{
+				label: `(${
+					document.querySelector("div.Header_timer__36MsP")?.textContent
+				}) Rejoindre la partie`,
+				url: `https://wolfy.fr/game/${gameId}`
+			}
+		];
+	} else {
+		presenceData.buttons[1] = {
+			label: `(${
+				document.querySelector("div.Header_timer__36MsP")?.textContent
+			}) Rejoindre la partie`,
+			url: `https://wolfy.fr/game/${gameId}`
+		};
+	}
+}
+
+async function addVisitProfilButton(
+	presenceData: PresenceData,
+	username: string
+) {
+	if (!(await presence.getSetting("visitProfileButton"))) return;
+
+	let label = `Visiter le profil de ${username}`;
+	if (label.length > 32) label = `${label.slice(0, 31)}…`;
```

Comment:
31 + 3 = 34

Reply:
It's not "..." it's "…" that is a single unicode character

Reply:
![image](https://user-images.githubusercontent.com/48710891/163954025-6f08d949-2502-4beb-b172-a0eab4dae421.png)